prod-build/deploy-docker-image.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+APP_VERSION=$(npm run env | grep ^npm_package_version= | cut -d "=" -f 2)
+IMAGE_NAME="ampnet/ampnet-crowdfunding-frontend:${APP_VERSION}"
+
+docker build --no-cache -f prod-build/Dockerfile -t "${IMAGE_NAME}" .
```

Comment:
Add tag: `latest`

Reply:
I've edited this script for deploying from a local machine only. Furthermore, I've extracted this logic inside GH action for deployment with the additional "latest" tag. This seems more secure if we split configurations, rather than using the same script with some flags. Maybe we'll change this in the future.